jest/env.js

```diff
@@ -36,7 +38,7 @@ global.test = (transformName, testFileName, options, fakeOptions) => {
     }
 
     expect(
-        (transform({path, source}, {jscodeshift}, options || {}) || "").trim()
+        (transform({path, fullPath, source}, {jscodeshift}, options || {}) || "").trim()
```

Comment:
I think options get passed to recast, fakeOptions are used to override local options.